gmt/datasets/tutorial.py

```diff
@@ -38,6 +38,27 @@ def load_japan_quakes():
     return data
 
 
+def load_tut_ship():
```

Comment:
Could you rename this function to `load_sample_bathymetry`? I really don't like the name `tut_ship`.

Reply:
Sounds good, done!